mscr-ui/src/modules/form/index.tsx

```diff
@@ -566,69 +568,67 @@ export default function FormModal({
                     ? SpinnerType.CrosswalkRegistrationModal
                     : modalType == ModalType.RegisterNewMscr
                       ? SpinnerType.CrosswalkCreationModal
-                      : SpinnerType.CrosswalkRevisionModal
-              }
+                      : SpinnerType.CrosswalkRevisionModal}
             ></SpinnerOverlay>
           )}
         </>
-        <div id={'modalTop'}></div>
-        <ModalTitle>
-          {contentType == Type.Schema
-            ? modalType == ModalType.RegisterNewFull
-              ? t('content-form.title.schema-register')
-              : modalType == ModalType.McsrCopy
-                ? t('content-form.title.schema-mscr-copy')
-                : t('content-form.title.schema-revision')
-            : modalType == ModalType.RegisterNewFull
-              ? t('content-form.title.crosswalk-register')
-              : modalType == ModalType.RegisterNewMscr
-                ? t('content-form.title.crosswalk-create')
-                : modalType == ModalType.McsrCopy
-                  ? t('content-form.title.crosswalk-mscr-copy')
-                  : t('content-form.title.crosswalk-revision')}
-        </ModalTitle>
-
-        {(modalType == ModalType.RegisterNewFull ||
-          modalType == ModalType.RevisionFull) &&
-          renderFileDropArea()}
+          <div id={'modalTop'}></div>
```

Comment:
The indentations are a bit all over the place, maybe run Prettier to get the formatting in order? I don't always remember to do that, but it would be a good habit to do when you've finished developing.

Reply:
formatting is fixed